.travis.yml

```diff
@@ -21,7 +21,7 @@ cache:
     - $HOME/.composer/cache
 
 before_install:
-  - ./travis/before_install.sh
+  - composer self-update
```

Comment:
@GrahamCampbell I'm not overly attached to it, but I've seen composer hit the >30 days warning so it's been a habit to add it privately. This may all have changed since, of course.